rfcs/text/0000-dss-deletion-daemon.md

```diff
@@ -0,0 +1,279 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Deletion and deprecation of data in the DCP
```

Comment:
Depreciation doesn't appear to be addressed in this RFC.

Reply:
will remove